sympy/matrices/expressions/determinant.py

```diff
@@ -32,14 +33,16 @@ def __new__(cls, mat):
     def arg(self):
         return self.args[0]
 
-    def doit(self, expand=False):
+    def doit(self, expand=False, **kwargs):
```

Comment:
I'm still not in favor of using `**kwargs`.

Reply:
Please see my comment below.